NPM RUN DEPLOY bug

Hi, 
I wanted to test fast but I can't deploy using npm (whole setup was initialized with npm). 
When typed npm run build I've got
`MacBook-Pro-koko:fast.test koko$ npm run deploy

> fast@0.0.1 deploy /Users/koko/Projects/fast.test
> yarn build && cd dist && surge
sh: yarn: command not found
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! fast@0.0.1 deploy: `yarn build && cd dist && surge`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the fast@0.0.1 deploy script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/koko/.npm/_logs/2020-10-04T22_10_08_146Z-debug.log`

--- a/README.md
+++ b/README.md
@@ -23,7 +23,7 @@ Using **Fast** is super simple. Simply `open the Fast folder` using your favorit
 **Note:** *We use [Yarn](https://yarnpkg.com/) for fast, reliable, and secure dependency management but feel free to use [NPM](https://www.npmjs.com/) if you prefer.*
 
 ### ⚠️ Are you on Windows?
-**Replace all instances of `$npm_execpath` in the package.json file with:**
+**Replace all instances of `$npm_execpath` in the [package.json](package.json) file with:**
 ```
 %npm_execpath%
 ```

--- a/README.md
+++ b/README.md
@@ -22,6 +22,12 @@ Using **Fast** is super simple. Simply `open the Fast folder` using your favorit
 
 **Note:** *We use [Yarn](https://yarnpkg.com/) for fast, reliable, and secure dependency management but feel free to use [NPM](https://www.npmjs.com/) if you prefer.*
 
+### ⚠️ Are you on Windows?
+**Replace all instances of `$npm_execpath` in the package.json file with:**
+```
+%npm_execpath%
+```
+
 ### Installing the necessary dependencies.
 ```
 yarn

--- a/package.json
+++ b/package.json
@@ -5,10 +5,10 @@
   "main": "index.js",
   "scripts": {
     "start": "parcel -p 3000 index.html",
-    "serve": "yarn start --open",
+    "serve": "\"$npm_execpath\" start --open",
     "build": "parcel build index.html",
-    "deploy": "yarn build && cd dist && surge",
-    "redeploy": "yarn build && cd dist && surge --domain",
+    "deploy": "\"$npm_execpath\" build && cd dist && surge",
+    "redeploy": "\"$npm_execpath\" build && cd dist && surge --domain",
     "teardown": "surge teardown"
   },
   "repository": {

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "fast",
-  "version": "0.0.1",
+  "version": "1.0.0",
   "description": "Develop, build, deploy, redeploy, and teardown frontend projects fast ⚡️",
   "main": "index.js",
   "scripts": {

